lib/es.js

```diff
@@ -0,0 +1,10 @@
+const en = require('./en');
+
+// See https://www.spanishdict.com/guide/spanish-punctuation
+const COMMON_PUNCTUATION = en.COMMON_PUNCTUATION + `¿¡«»`;
+const COMMON_SYMBOLS = en.COMMON_SYMBOLS + `€`;
```

Comment:
Do you think there are use cases for splitting en and es rules like this? I wonder if we can just include these characters in the set and have common sets.

Reply:
I don't know if our particular product has any, though I can imagine that perhaps a Spanish-speaker/English-learner might by habit include guillemots in their English answer and we'd want to mark that wrong, but if they used quotes we'd accept it.

I can contrive some more general cases, like if some quiz had a question where answers were amounts of money. In an American English quiz context, `$` would probably not matter (500 and $500 might both be acceptable answers), but a Euro sign is out of place and thus it matters. 

How about if there's also a combined list that merges all language lists, available on `Equivalent.COMMON_PUNCTUATION`?